### DISCUSSION TITLE
Question: i think lime encode project.xml wrong

### DISCUSSION THREAD

**Discussion by HumanKR** [FunkinCrew/Funkin#1513] [Q&A]
## What is your question?
i making some translation and i was edit some project.xml
i changed app title, but characters is... cracked? broke?
idk so i borrow some screenshots
![image](https://user-images.githubusercontent.com/45139720/113793833-2b211e00-9784-11eb-84c1-23f1c04df688.png)
that app title(Korean) into... this
![image](https://user-images.githubusercontent.com/45139720/113793873-47bd5600-9784-11eb-823f-82a222aa1d25.png)
that's.. weird..
can i fix it? it seems like Japanese also have problem about that
or utf-8 is not support it?

**Comment by TentaRJ** [comment]
UTF-8 is a bit weird...

**Comment by HumanKR** [comment]
noooooooou
it is impossible to fix this?

**Comment by TentaRJ** [comment]
> noooooooou
> 
> it is impossible to fix this?

I'll try to figure it out. Could you send the translated text here in both English and Korean text?

**Comment by Two-Ai** [comment]
I tried finding a minimum repro and I managed to strip it down to a project title, an empty `Main` and a `flixel` dependency.
I haven't tried working around the flixel dependency so idk if the issue is in flixel, open-fl or haxe, but it's not a FNF issue.
https://github.com/Two-Ai/Funkin/tree/unicode-title-min-repro

**Comment by HumanKR** [comment]
> > noooooooou
> > it is impossible to fix this?
> 
> I'll try to figure it out. Could you send the translated text here in both English and Korean text?

프라이데이 나이트 펑킨
that means Friday night funkin
or shorter one
"test" in Korean, "테스트".
i hope there are good results
